src/models/index.js

```diff
@@ -19,7 +19,7 @@ if (config.use_env_variable) {
 fs
   .readdirSync(__dirname)
   .filter(file => {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js')
+    return (file.indexOf('.') !== 0) && (file !== basename) && !file.endsWith('.test.js') && file.endsWith('.js')
```

Comment:
`file.endsWith('.js')` will filter empty files and your base name, so the first two conditions are (IMHO) no needed at all.

Reply:
Actually, if I have two files `.doNotIncludeAsAModel.js` and `index.js` they *will not* be filtered out by `file.endsWith('.js')`, since them both end in `.js`.

Reply:
Yes, but you don't have any logic to filter these "fake models". Now, in order for a file to be a "model file" it has do end with `.js` but not with `.test.js`. At least, that's what you're doing now.

If you need further logic, go ahead and add it.

Reply:
I'm not sure I follow, what do you mean about further logic? Should I abstract the filters to their own functions (`isIndexFile`, `isTestFile`)?

The first filter (`file.indexOf('.') !== 0`) is not needed. I think it was added by Sequelize CLI when we were in Block 3. I'll remove it.

I think the second filter is definitely needed, otherwise `index.js` would be imported by Sequelize and generate an error.

Reply:
Yes, you're right. Not need to write separate functions I guess.